examples/minimal-mdns/advertiser.cpp

```diff
@@ -81,6 +86,10 @@ bool HandleOptions(const char * aProgram, OptionSet * aOptions, int aIdentifier,
         {
             gOptions.advertisingMode = AdvertisingMode::kCommisioning;
         }
+        else if ((strcmp(aValue, "commisionable") == 0) || (strcmp(aValue, "d") == 0))
```

Comment:
the 'd' is awkward. Please remove 'c' from above instead since 'c' seems ambigous for commisiong/commisionable

Reply:
I used 'd' because for commisionable we advertise as _chipd while for commisionable we advertise as _chipc and hence 'c' was used.